openmetadata-docs/content/v1.2.x/deployment/security/enable-jwt-tokens.md

```diff
@@ -50,8 +50,8 @@ To enable JWT token generation. Please add the following to the OpenMetadata ser
 
 ```yaml
 jwtTokenConfiguration:
-  rsapublicKeyFilePath: ${RSA_PUBLIC_KEY_FILE_PATH:-"/openmetadata/conf/public_key.der"}
-  rsaprivateKeyFilePath: ${RSA_PRIVATE_KEY_FILE_PATH:-"/openmetadata/conf/private_key.der"}
+  rsapublicKeyFilePath: ${RSA_PUBLIC_KEY_FILE_PATH:-"/opt/openmetadata/conf/public_key.der"}
+  rsaprivateKeyFilePath: ${RSA_PRIVATE_KEY_FILE_PATH:-"/opt/openmetadata/conf/private_key.der"}
```

Comment:
@akash-jain-10 should we be changing this for all the versions or just 1.2.x?

Reply:
Thanks for reviewing this. 

This is the only place which requires this change! I did check all the contents for 1.2.X and the snippet is only used here!

Reply:
ok thanks for confirming 👍